dotcom-rendering/src/lib/liveblogAdSlots.test.ts

```diff
@@ -98,6 +173,7 @@ describe('shouldDisplayAd', () => {
 			totalBlocks,
 			numAdsInserted,
 			numPixelsWithoutAdvert,
+			false,
 		);
 
 		expect(result).toBeFalsy();
```

Comment:
Not a blocker, but is it worth testing this for both mobile and desktop, for consistency with the test for inserting an ad after the first block?

Reply:
Yeah good shout, I missed this one. I'll also refactor the file for testing both mobile and desktop as I've remembered a tidier way to do it.